### ISSUE TITLE
Light Mode on our Auto-generated docs

### ISSUE THREAD

**Issue by joshenlim** [supabase/supabase#4330]
# Chore

![image](https://user-images.githubusercontent.com/19742402/144804081-e326f82e-b794-4337-8f3a-919cb4e90e5a.png)

**Comment by harshcut** [comment]
I am currently trying to fix this (#4330) issue. But I can't seem to reflect the changes made in `studio` directory when running `docker-compose up`. I followed the documentation given on [Self Hosting](https://supabase.com/docs/guides/hosting/docker).

```console
git clone --depth 1 https://github.com/supabase/supabase
cd .\supabase\docker\
docker-compose up
```

I even tried running `studio` independently, but that resulted in a compile error.

```console
cd .\supabase\studio\
npm install
npm run dev
```

<details>
<summary>Compilation Error</summary>

```console
wait  - compiling...
error - ./styles/code.scss
Global CSS cannot be imported from files other than your Custom <App>. Due to the Global nature of stylesheets, and to avoid conflicts, Please move all first-party global CSS imports to pages/_app.js. Or convert the import to Component-Level CSS (CSS Modules).
Read more: https://nextjs.org/docs/messages/css-global
Location: pages\_app.tsx
```

</details>

**Comment by MildTomato** [comment]
@harshcut 
It's probably easier to stick to tailwind classes as I think we are moving away from using imported CSS files.

**Comment by joshenlim** [comment]
@harshcut the best way to work on studio is probably to run your services via docker, then run studio separate via npm run dev (probably need to close the studio container in docker)

Are you still running into this compilation issue? If not could i check what node version you're on?

**Comment by harshcut** [comment]
> @harshcut the best way to work on studio is probably to run your services via docker, then run studio separate via npm run dev (probably need to close the studio container in docker)
> 
> Are you still running into this compilation issue? If not could i check what node version you're on?

The issue was fixed in #4416.

**Comment by MildTomato** [comment]
@harshcut ah i see what you mean now - can see the auto docs is using css rather than tailwind for a lot of it.
feel free to just implement it how you like for now. 

you could always use css like this

```css

.docs-class {
 /* light theme styles */
}

.dark .docs-class {
 /* dark theme styles */
}

```

styles could be added into [styles/ui.css](https://github.com/supabase/supabase/blob/238b3430f5b6f5e0479fb3218a5c76158e1149e4/studio/styles/ui.scss#L197)
